src/filters.ts

```diff
@@ -0,0 +1,43 @@
+import * as semver from "semver"
+import {MigrationMode} from "./settings"
+
+const logger = require('pino')()
+
+export function filterSourceVersions(packageName: string,
+                                     sourcePackument: Record<string, any>,
+                                     migrationMode: MigrationMode) {
+  const allSourceVersions = sourcePackument ? (sourcePackument.versions ? Object.keys(sourcePackument.versions) : []) : []
+  logger.debug(`[${packageName}] source versions: ${allSourceVersions}`)
+
+  let filteredSourceVersions = []
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263162)
```suggestion
  let filteredSourceVersions = [];
```